src/main/java/lc/captchas/PoppingCharactersCaptcha.java

```diff
@@ -36,16 +36,18 @@
       advances.add(currX);
       currX += font.getStringBounds(String.valueOf(c), frc).getWidth();
       currX += spacing;
-    };
+    }
+    ;
```

Comment:
I suppose this is being done by the formatter. Let's either disable auto-format on compile or commit such changes in separate PRs. This sort of thing will cause merge conflicts later.

Please undo the changes to this file.